src/posthog-surveys.ts

```diff
@@ -75,10 +76,20 @@ export class PostHogSurveys {
     }
 
     onRemoteConfig(response: RemoteConfig) {
-        this._decideServerResponse = !!response['surveys']
-        logger.info(`decideServerResponse set to ${this._decideServerResponse}`)
+        // only load surveys if they are enabled and there are surveys to load
+        const surveys = response['surveys']
+        if (isArray(surveys)) {
+            this._hasSurveys = surveys.length > 0
+            this._decideServerResponse = true
+        } else if (isBoolean(surveys)) {
+            this._hasSurveys = surveys
+            this._decideServerResponse = true
+        }
 
-        this.loadIfEnabled()
+        if (this._decideServerResponse) {
+            logger.info(`decideServerResponse set to ${this._decideServerResponse}, hasSurveys: ${this._hasSurveys}`)
+            this.loadIfEnabled()
+        }
```

Comment:
with this i feel like `_decideServerResponse` is not really needed anymore? since it's only used to check that. unless i'm missing something? and then we can just use `hasSurveys` which is more meaningful too

Reply:
its more about the logging.
decide not loaded != not having surveys

Reply:
i did the logging elsewhere and removed the _decideServerResponse anyway